src/agent/src/config.rs

```diff
@@ -24,6 +24,7 @@ const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
 const CONTAINER_POLICY_FILE: &str = "agent.container_policy_file";
+const HTTPS_PROXY: &str = "agent.https_proxy";
```

Comment:
can we need no_proxy to pull images from internal registries?

Reply:
Yes, if we pull image from both internal and external registries, we need no_proxy, fixed now.